org.mwc.cmap.plotViewer/src/org/mwc/cmap/plotViewer/editors/CorePlotEditor.java

```diff
@@ -359,7 +363,8 @@ public void addThisLayer(final Layer theLayer) {
 				if (theLayer instanceof ExternallyManagedDataLayer) {
 					final ExternallyManagedDataLayer dl = (ExternallyManagedDataLayer) theLayer;
 					if (dl.getDataType().equals(MWC.GUI.Shapes.ChartBoundsWrapper.WORLDIMAGE_TYPE)) {
-						final GeoToolsLayer gt = new WorldImageLayer(dl.getName(), dl.getFilename());
+						
+						final GeoToolsLayer gt = new WorldImageLayer(dl.getName(), getFixedFilePath(dl.getFilename()));

```

Comment:
@ayeshare  - your new method can return null (if the file can't be found).  I don't think we should create a `WorldImageLayer` if the file wasn't found.